### PR TITLE
feature-19

### PR DESCRIPTION
**Pull request by samuskitchen**
## Summary
- The information of the logged in User was added
## Trello Card
https://trello.com/c/8BIKLTpk/27-acceder-a-la-informaci%C3%B3n-del-user-logueado

### CODE REVIEW

File: src/main/java/wolox/training/controllers/SecurityController.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+public class SecurityController {
+
+    private final UserRepository userRepository;
+
+    public SecurityController(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @GetMapping("/username")
+    @ApiOperation(value = "Giving an author, return one book", response = Book.class)
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/controllers/SecurityController.java]: Oops, careful with the copy paste 😅

File: src/main/java/wolox/training/controllers/SecurityController.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Success"),
+            @ApiResponse(code = 401, message = "Not Authorized"),
+            @ApiResponse(code = 403, message = "Access forbidden"),
+            @ApiResponse(code = 404, message = "User Not Found"),
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/controllers/SecurityController.java]: I think this status code is not possible on this endpoint

**Review by Jimenarosello — COMMENTED** [review]
Nice job! I just left a few comments :)

File: src/main/java/wolox/training/controllers/SecurityController.java (hunk 1 above)
Comment (samuskitchen) [reply src/main/java/wolox/training/controllers/SecurityController.java]: jajajaja

**Review by JaimLus — APPROVED** [review]
Nice job!

**Pull request merged**